pyartifactory/models/repository.py

```diff
@@ -41,6 +40,8 @@ class PackageTypeEnum(str, Enum):
     vagrant = "vagrant"
     conda = "conda"
     cargo = "cargo"
+    terraform = "terraform"
+    terraformbackend = "terraformbackend"

```

Comment:
I cannot see terraformbackend in the [artifactory api](https://www.jfrog.com/confluence/display/JFROG/Repository+Configuration+JSON). Can you point me to it please ?